core/src/main/java/apoc/export/cypher/MultiStatementCypherSubGraphExporter.java

```diff
@@ -229,7 +229,7 @@ private void exportSchema(PrintWriter out) {
     }
 
     private List<String> exportIndexes() {
-        return db.executeTransactionally("CALL db.indexes()", Collections.emptyMap(), result -> result.stream()
+        return db.executeTransactionally("SHOW INDEXES", Collections.emptyMap(), result -> result.stream()
```

Comment:
can we presume that the result columns have not changed between these two calls?

Reply:
Nope, there is one change: "provider" is replaced by "indexProvider"

Reply:
But from what I could see in this method we never care about that field so I think we are good.